crates/wmemcheck/fuzz/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "wmemcheck-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2021"
+
+[package.metadata]
+cargo-fuzz = true
+
+[dependencies]
+libfuzzer-sys = { version = "0.4.0", features = ["arbitrary-derive"] }
+rand="0.3.14"
+
+[dependencies.wmemcheck]
+path = ".."
+
+# Prevent this from interfering with workspaces
+[workspace]
+members = ["."]
```

Comment:
Could this be added to the overall workspace in this repository? (there's a few other examples of crates that have their own fuzzers which are also part of the main workspace)

Reply:
I've gone ahead and deleted the `fuzz` crate -- I think we had talked about this earlier but missed it, as the fuzz targets are out-of-date anyway.